src/Views/AppListView.vala

```diff
@@ -231,6 +231,21 @@ namespace AppCenter.Views {
                 return a_is_driver ? - 1 : 1;
             }
 
+            string? a_timestamp = row1.get_timestamp ();
+            string? b_timestamp = row2.get_timestamp ();
+
```

Comment:
Suggest string a_timestamp = row1.get_timestampe () ?? "" and same for b_timestamp, in order to avoid possibly comparing null with null.